lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1733,7 +1738,7 @@ public Response makeRequestBulk(
             final File jsonData) 
             throws IOException, IndexingConflictException {
         try (InputStream is = new FileInputStream(jsonData)) {
-            return makeRequest(reqType, "/" + indexName + "/_bulk", Collections.emptyMap(),
+            return makeRequest(reqType, "/" + indexName + "/_bulk?refresh", Collections.emptyMap(),
```

Comment:
Should document that this method now causes an immediate refresh. AFAICT it's only used in indexObjects() so the change doesn't affect any other operations.

Reply:
Done.